sdk/appconfiguration/azure-appconfiguration/CHANGELOG.md

```diff
@@ -1,16 +1,10 @@
 # Release History
 
-## 1.4.0 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.3.1 (2022-02-07)
```

Comment:
What's the target version number?

1.4.0b1? (I asked this because I see you change the file below to "Development Status :: 4 - Beta".

Also if we update the version number in changelog, we also need to update it in version.py.

Reply:
(btw, dropping 2.7 support should be a minor version bump.)

Reply:
Thanks for catching! I think it should be 1.4.0b1 because there just has some dependency updates and transport type updates. Any thoughts?